### DISCUSSION TITLE
Do existing RPort server installation need to be removed to use openrport ?

### DISCUSSION THREAD

**Discussion by ebym** [openrport/openrport#34] [Q&A]
I'm running RPort version 0.9.12, do i have to uninstall existing rport to use openrport ?. If so, Is there a migration script for this ?.

Thanks.

**Comment by alhdo** [comment]
Yes to have a clean install you should uninstall it since it not the same original binary. We are working on a script to make it automatically. but make sure you update your client first to openrport 0.9.14. The new version will connect to the old version.

**Comment by ebym** [comment]
Thanks, what are the current steps ?.

Upgrade client to 0.9.14 and clean install openrport ?.

Wouldn't clean install of server remove all the existing logs ?.

**Comment by alhdo** [comment]
Hello, 
Here is a step by step guide on how to go with it:

1 - In your current server change the pairing url to https://pairing.openrport.com, check the readme of the repo to see how
2 - Upgrade your client to 0.9.14, check the documentation to see how [Knowledge Base](https://kb.openrport.com)
3 - Make a backup of your current server to be safe. Save your fingerprint. You should be using FQDN otherwise you won't be able to recover your device (If you don't have your own Fully Qualified Domain) I will advice you to do a clean install. 
4 - Do a clean install of OpenRport server 0.9.14 and update the conf file to put back your fingerprint.
5 - Devices should reconnect automatically

Yes a clean install will remove all the log but since the server keep the same FQDN and fingerprint. You should also copy the clients database to the new install. Client will get connected automatically.

Please let me know if you have any more question.

**Comment by ebym** [comment]
Many thanks. Will wait for your migration script to be safe.

Assume the migration script you are working on will backup everything(logs, databases, etc....), unistall rport, perform clean install of openrport and restore all the settings, logs, databases, etc...
